plenum/test/node_catchup_with_3pc/test_stashing_checkpoints_after_view_change.py

```diff
@@ -105,20 +104,20 @@ def test_checkpoints_after_view_change(tconf,
     )
     looper.run(
         eventually(check_last_ordered_3pc_on_backup, [lagging_node],
-                   (1, num_reqs + 1))
+                   (1, num_reqs))
     )
 
     # check that checkpoint is stabilized for master
-    looper.run(eventually(check_for_instance, [lagging_node], 0, check_stable_checkpoint, 10))
+    looper.run(eventually(check_for_instance, txnPoolNodeSet[:2], 0, check_stable_checkpoint, 10))
```

Comment:
Why change check to something similar to "rest of the nodes" instead of lagging node?

Reply:
fixed